nifi-nar-bundles/nifi-standard-services/nifi-hbase_1_1_2-client-service-bundle/nifi-hbase_1_1_2-client-service/src/main/java/org/apache/nifi/hbase/HBase_1_1_2_ClientService.java

```diff
@@ -171,6 +188,15 @@ protected Connection createConnection(final ConfigurationContext context) throws
             }
         }
 
+        UserGroupInformation.setConfiguration(hbaseConfig);
+
+        if (UserGroupInformation.isSecurityEnabled()) {
+            LOG.info("SECURITY IS ENABLED");
+            UserGroupInformation.loginUserFromKeytab(context.getProperty(KERBEROS_PRINCIPAL).getValue(),
```

Comment:
Can we try/catch IOException when logging in the user in case of an exception. It can be as simple as logging the error being with the keytab and then bubble up the exception.